docs/DatabaseCatalogAPI.yaml

```diff
@@ -311,13 +341,13 @@ components:
         bounding_box:
           $ref: "#/components/schemas/BoundingBox"
 
-    FeedStatus:
+    ElementStatus:
       description: > 
-        Describes status of the feed. Should be one of 
-          * `active` Feed should be used in public trip planners.
-          * `deprecated` Feed is explicitly deprecated and should not be used in public trip planners.
-          * `inactive` Feed hasn't been recently updated and should be used at risk of providing outdated information.
-          * `development` Feed is being used for development purposes and should not be used in public trip planners.
+        Describes status of the element (Feed or Dataset). Should be one of 
```

Comment:
Can you explain why status would apply to both dataset + feed? I think this is a feed level property.